getting-started/templates/systemlink-values.yaml

```diff
@@ -1029,29 +1029,36 @@ argoworkflows:
 ##
 nbexecservice:
   maxNumberOfWorkflowsToSchedule: *workflowParallelism
-  ## Configure S3 access.
+  ## Configures file storage settings.
   ##
-  s3:
-    ## Secret name for S3 credentials.
-    ##
-    secretName: "nbexecservice-s3-credentials"
-    ## The name of the S3 bucket that the service should connect to.
-    ##
-    bucket: "systemlink-executions"
-    ## S3 connection scheme.
-    ##
-    scheme: *s3Scheme
-    ## Set this value to connect to an external S3 instance.
-    ##
-    host: *s3Host
-    ## Set this value to connect to an S3 instance which is internal to the cluster. Ignored if host is set.
+  storage: 
+    ## Specifies which storage to use. Supports "S3".
     ##
-    service: *s3ServiceName
-    ## S3 Port
-    port: *s3Port
-    ## S3 Region
+    type: "s3"
+    ## Configures S3 access.
     ##
-    region: *s3Region
+    s3:
+      ## Secret name for S3 credentials.
+      ##
+      secretName: "nbexecservice-s3-credentials"
+      ## The name of the S3 bucket that the service should connect to.
+      ##
+      bucket: "systemlink-executions"
+      ## S3 connection scheme.
+      ##
+      scheme: *s3Scheme
+      ## Set this value to connect to an external S3 instance.
+      ##
+      host: *s3Host
+      ## Set this value to connect to an S3 instance which is internal to the cluster. Ignored if host is set.
```

Comment:
Suggestion: Set this value to connect to an S3 instance that is internal to the cluster. The system ignores the value if the host is set.